ZocBuild.Database.Application/MainWindow.xaml.cs

```diff
@@ -111,7 +112,7 @@ private void btBuild_Click(object sender, RoutedEventArgs e)
                     var fileSystem = new FileSystem();
                     var db = dbSetting.Create(connection, transaction);
                     var gitProcess = new ExternalProcess(pathToGit.FullName);
-                    var dvcsScriptRepo = new GitScriptRepository(dbSetting.ScriptsPath, dbSetting.ServerName, dbSetting.DatabaseName, gitProcess, fileSystem, sqlParser, false);
+                    var dvcsScriptRepo = new GitScriptRepository(dbSetting.ScriptsPath, dbSetting.ServerName, dbSetting.DatabaseName, gitProcess, fileSystem, sqlParser, new ConsoleLogger(SeverityLevel.Warning), false);

```

Comment:
Right, if you were to do it, `WpfLogger` would be a class that wraps a WPF `Window` and delegates the messages to append to a read-only `TextBox`.  The DevDataRestore.Messaging.Windows application has something like that, I think.